b2sdk/_v3/__init__.py

```diff
@@ -189,19 +182,30 @@
 from b2sdk.sync.policy_manager import POLICY_MANAGER
 from b2sdk.sync.report import SyncFileReporter
 from b2sdk.sync.report import SyncReport
-from b2sdk.sync.scan_policies import DEFAULT_SCAN_MANAGER
-from b2sdk.sync.scan_policies import IntegerRange
-from b2sdk.sync.scan_policies import RegexSet
-from b2sdk.sync.scan_policies import ScanPoliciesManager
-from b2sdk.sync.scan_policies import convert_dir_regex_to_dir_prefix_regex
 from b2sdk.sync.sync import KeepOrDeleteMode
 from b2sdk.sync.sync import Synchronizer
-from b2sdk.sync.sync import zip_folders
 from b2sdk.sync.encryption_provider import AbstractSyncEncryptionSettingsProvider
 from b2sdk.sync.encryption_provider import BasicSyncEncryptionSettingsProvider
 from b2sdk.sync.encryption_provider import ServerDefaultSyncEncryptionSettingsProvider
 from b2sdk.sync.encryption_provider import SERVER_DEFAULT_SYNC_ENCRYPTION_SETTINGS_PROVIDER
 
+# scan
+
+from b2sdk.scan.exception import EnvironmentEncodingError
+from b2sdk.scan.exception import InvalidArgument
+from b2sdk.scan.folder import AbstractFolder
+from b2sdk.scan.folder import B2Folder
+from b2sdk.scan.folder import LocalFolder
+from b2sdk.scan.folder_parser import parse_folder
```

Comment:
I think in v2 both `parse_sync_folder` and `parse_folder` should be available and there should be a test that makes sure we have it